webapp/lib/app/configurator/standard_messages/controller.dart

```diff
@@ -68,6 +70,27 @@ class StandardMessagesCategoryData extends Data {
   }
 }
 
+class StandardMessagesCategoriesReorderData extends Data {
+  List<String> categoryIds;
+  StandardMessagesCategoriesReorderData(this.categoryIds);
```

Comment:
see my comment in the other PR - I think we should pass here only the data that describes the reordering action (so id and index), and not the computed list of ids.

and also the logic of reordering those messages should be moved in this PR (in the controller), which should then call various methods for adjusting the accordion (if possible in place)